app/grandchallenge/reader_studies/models.py

```diff
@@ -827,6 +835,13 @@ def assign_permissions(self):
 
     class Meta:
         ordering = ("order", "created")
+        constraints = [
+            models.UniqueConstraint(
```

Comment:
While we're at it, should we also add a uniqueness constraint for `order` and `reader_study`? At the moment, you can have multiple display sets with the same order.

Reply:
I think that is not too badly needed. Order uniqueness would require a migration and is out of scope =P.